openpype/vendor/python/common/capture.py

```diff
@@ -760,6 +760,8 @@ def _applied_viewport_options(options, panel):
         # Try to set as much as possible of the state by setting them one by
         # one. This way we can also report the failing key values explicitly.
         for key, value in options.items():
+            if key == "reloadTextures":
+                continue
```

Comment:
We should not be updating the `capture` library for this. We should instead not be passing this flag to the `capture` call to begin with. This is too specific to our implementation to require changing an external dependency.

So we should pop this key before passing the preset to `capture`